.github/workflows/tests.yml

```diff
@@ -53,39 +53,29 @@ jobs:
 
   im-node-tests:
     name: IM Node Tests
-    runs-on: macos-latest
+    runs-on: ubuntu-latest
     timeout-minutes: 30
     strategy:
       matrix:
         node-version: ["18.15.0"]
     steps:
       - name: Checkout code
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
         with:
           fetch-depth: 0
 
       - name: Setup Node.js
-        uses: actions/setup-node@v2
+        uses: actions/setup-node@v3
         with:
           node-version: ${{ matrix.node-version }}
 
+      - name: Run Era Test Node
+        uses: dutterbutter/era-test-node-action@latest
```

Comment:
Neat, let's move this action to the `matter-labs` org

Reply:
We discussed this yesterday, its intentionally outside of the org. @idea404